tomato-timer/public/index.html

```diff
@@ -1,6 +1,19 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <script src="https://www.gstatic.com/firebasejs/4.1.1/firebase.js"></script>
+      <script>
+        // Initialize Firebase
+        var config = {
+          apiKey: "AIzaSyCZytKSL4snOizTFVq_lNyA3bRKHUqrvyw",
```

Comment:
No API Keys in the repo! [Environment variables](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-custom-environment-variables)